.ci/containers/terraform-vcr-tester/check_membership.sh

```diff
@@ -7,7 +7,7 @@ USER=$(curl -H "Authorization: token ${GITHUB_TOKEN}" \
   "https://api.github.com/repos/GoogleCloudPlatform/magic-modules/issues/${PR_NUMBER}" | jq -r .user.login)
 
 # Only run tests for safe users
-if $(echo $USER | fgrep -wq -e ndmckinley -e danawillow -e emilymye -e megan07 -e paddycarver -e rambleraptor -e SirGitsalot -e slevenick -e c2thorn -e rileykarson -e melinath); then
+if $(echo $USER | fgrep -wq -e ndmckinley -e danawillow -e emilymye -e megan07 -e paddycarver -e rambleraptor -e SirGitsalot -e slevenick -e c2thorn -e rileykarson -e melinath -e scottsuarez); then
 	echo "User is on the list, not skipping."
 else
 	echo "Checking GCP org membership"
```

Comment:
I'm in gcp org so this shouldn't be necessary. I suppose it wouldn't hurt though

Reply:
@slevenick can we just remove all the googlers from this list? that makes sense, right?

Reply:
Yeah that makes sense. @ScottSuarez have builds been triggering automatically for you?

Reply:
yep ! They've been triggering for me